VideoLocker/res/layout/row_course_list.xml

```diff
@@ -156,7 +159,8 @@
             android:layout_alignParentRight="true"
             android:layout_alignTop="@id/schoolcode_layout"
             android:layout_below="@id/course_name"
-            android:layout_marginRight="10dp" >
+            android:layout_marginRight="10dp"
+            android:layout_toRightOf="@id/schoolcode_layout">
```

Comment:
My assumption is that right most view will not have too much variation in its width.